### PR TITLE
setup.py

### PR DESCRIPTION
**Pull request by vrutkovs**
This file is required so that other packages (ota-autoreport) could add it to Pipenv file

### CODE REVIEW

**Comment by vrutkovs** [comment]
cc @wking

**Comment by vrutkovs** [comment]
/hold

This doesn't really work yet

**Comment by vrutkovs** [comment]
/hold cancel

Ready for review
/cc @wking @LalatenduMohanty

File: cluster_support_bot/hydra.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import errors
+from . import errors
```
Comment (wking) [inline cluster_support_bot/hydra.py:1]: [PEP 8][1] would probably prefer:

```python
import datetime

import requests

from . import errors
```

But I'm fine as this stands too.

[1]: https://www.python.org/dev/peps/pep-0008/#imports

**Pull request merged**